lib/Dialect/Seq/SeqPasses.cpp

```diff
@@ -16,87 +16,3 @@
 using namespace mlir;
 using namespace circt;
 using namespace seq;
-
```

Comment:
You can just delete this file. No point in keeping it around if there's no code in it.

Reply:
Agreed